### ISSUE TITLE
Different softmax functions for prediction and soft label

### ISSUE THREAD

**Issue by dentou** [HikariTJU/LD#9]
Hello, may I ask why you used `log_softmax` for prediction but `softmax` for soft label? Please see the code lines below.
https://github.com/HikariTJU/LD/blob/f94c1d31cba50c83cf0d2182ad0f4782e019f850/mmdet/models/losses/gfocal_loss.py#L79-L84

And here is the equation in your paper:
![image](https://user-images.githubusercontent.com/27295564/118273085-c912b180-b4c3-11eb-8445-af683841df05.png)

Best regards

**Comment by HikariTJU** [comment]
According to [pytorch docs:](https://pytorch.org/docs/stable/generated/torch.nn.KLDivLoss.html)

> As with NLLLoss, the input given is expected to contain log-probabilities and is not restricted to a 2D Tensor. The targets are interpreted as probabilities by default

**Comment by dentou** [comment]
> According to [pytorch docs:](https://pytorch.org/docs/stable/generated/torch.nn.KLDivLoss.html)
> 
> > As with NLLLoss, the input given is expected to contain log-probabilities and is not restricted to a 2D Tensor. The targets are interpreted as probabilities by default

Thanks a lot for your quick response!
